test/ec_driver.cpp

```diff
@@ -71,30 +97,42 @@ void create_coding_files(
     quadiron::fec::FecCode<T>* fec,
     bool operation_on_packet = false)
 {
-    char filename[1024];
+    std::string filename;
     std::vector<std::istream*> d_files(fec->n_data, nullptr);
     std::vector<std::ostream*> c_files(fec->n_outputs, nullptr);
     std::vector<std::ostream*> c_props_files(fec->n_outputs, nullptr);
     std::vector<quadiron::Properties> c_props(fec->n_outputs);
 
     for (unsigned i = 0; i < fec->n_data; i++) {
-        snprintf(filename, sizeof(filename), "%s.d%d", prefix, i);
+        filename = get_filename(prefix, 'd', data_zpad, i, "");
         if (vflag)
             std::cerr << "create: opening data " << filename << "\n";
         d_files[i] = new std::ifstream(filename);
+        if (d_files[i]->fail()) {
+            std::cerr << "Exception opening " << filename << "\n";
```

Comment:
It would be nice to have different error message for the different case (so that looking at the error message, you know where piece of code failed).
e.g:

- l.112: "Exception opening data file (RO) for creation "
- l.124: "Exception opening coding file for creation "
- l.133: "Exception opening coding props for creation "
- l.187: "Exception opening data file for checking "
- l.194: "Exception opening data file (RO) for checking "
- l.211: "Exception opening coding file for checking "
- l.223: "Exception opening coding props for checking "